src/main/java/io/appium/java_client/serverevents/TimedEvent.java

```diff
@@ -0,0 +1,13 @@
+package io.appium.java_client.serverevents;
+
+import java.util.List;
+
+public class TimedEvent {
```

Comment:
I would still consider using lombok. Otherwise  I would prefer removing this dependency completely if it cannot be used